jest/setup.js

```diff
@@ -0,0 +1 @@
+// Global setup here.
```

Comment:
We don't really have tests yet which I didn't realize until I set this up, I assume we'll want to use this when we do, so leaving.